v3/zlint.go

```diff
@@ -76,7 +75,7 @@ func LintRevocationList(r *ox509.RevocationList) *ResultSet {
 //
 // If registry is nil then the global registry of all lints is used and this
 // function is equivalent to calling LintRevocationListEx(r).
```

Comment:
Thanks to the correction above I realized this should be `LintRevocationList(r)`